src/Tizen.Multimedia/AudioManager/AudioDevice.cs

```diff
@@ -104,9 +107,252 @@ public bool IsRunning
                     ThrowIfError("Failed to get the running state of the device");
 
                 return isRunning;
+			}
+        }
+
+        /// <summary>
+        /// Gets the device's supported sample formats.
+        /// </summary>
+        /// <returns>An IEnumerable&lt;AudioSampleFormat&gt; that contains supported sample formats.</returns>
+        /// <remarks>
+        /// This device should be <see cref="AudioDeviceType.UsbAudio"/> type and <see cref="AudioDeviceIoDirection.Output"/> direction.
+        /// </remarks>
+        /// <exception cref="ArgumentException">This device is not valid or is disconnected.</exception>
+        /// <exception cref="InvalidOperationException">This device is not <see cref="AudioDeviceType.UsbAudio"/> type or is not <see cref="AudioDeviceIoDirection.Output"/> direction.</exception>
+        /// <since_tizen> 5 </since_tizen>
+        public IEnumerable<AudioSampleFormat> GetSupportedSampleFormats()
+        {
+            Interop.AudioDevice.GetSupportedSampleFormats(_id, out IntPtr formats, out uint numberOfElements).
+                ThrowIfError("Failed to get supported sample formats");
+
+            return RetrieveFormats();
```

Comment:
I think it's good to comply with the same style GetConnectedDevices() in AudioManager.cs.
We can take advantage of 'local' function as well.

Reply:
I ask again, why do you use 'local' function in here and GetConnectedDevices()?

What is 'local' function adevantage?

Reply:
As I said before, I think it's better to extract local functions to method. Because, the simillar logic is in GetSupportedSampleRates() method as local function.

How about extract it as generic method? (after remove converting logic. please refer below comment.)

Reply:
the advantage is as below.
"Local functions make the intent of your code clear. Anyone reading you code can see that the method is not callable except by the containing method. For team projects, they also make it impossible for another developer to mistakenly call the method directly from elsewhere in the class or struct."

Reply:
I don't feel to extract method as local function, because `GetSupportedSampleFormats` method is obviously simple.
But, I respect your opinion